metadatastore/mds.py

```diff
@@ -656,6 +657,13 @@ def insert_descriptor(self, run_start, data_keys, time, uid, **kwargs):
                                            data_keys=data_keys,
                                            time=time, uid=uid,
                                            **kwargs)
+    def _sanitize_np(self, val):
```

Comment:
Since this does not need access to `self` I'd suggest making it top-level function rather than a method. It's easier to test that way.